src/grid/PlanGridView.php

```diff
@@ -110,6 +110,16 @@ public function columns(): array
                     ],  
                 ]),
             ],
+            'parent_plan' => [
+                'format' => 'raw',
+                'value' => function (Plan $model): string {
+                    if (empty($model->parent_id)) {
+                        return '';
+                    }
+                    $parent = Plan::findOne($model->parent_id);
```

Comment:
This will cause 25 API queries to render a page in a grid.

Use [Eager Loading](https://www.yiiframework.com/wiki/834/relational-query-lazy-loading-and-eager-loading-in-yii-2-0) on the model level instead.